urdf2webots/writeProto.py

```diff
@@ -155,18 +155,39 @@ def URDFLink(proto, link, level, parentList, childList, linkList, jointList, sen
             proto.write((level + 1) * indent + 'physics Physics {\n')
             proto.write((level + 2) * indent + 'density -1\n')
             proto.write((level + 2) * indent + 'mass %lf\n' % link.inertia.mass)
-            if link.inertia.ixx > 0.0 and link.inertia.iyy > 0.0 and link.inertia.izz > 0.0:
-                proto.write((level + 2) * indent + 'centerOfMass [ %lf %lf %lf ]\n' % (link.inertia.position[0],
+            #  if link.inertia.position != [0.0, 0.0, 0.0]:
```

Comment:
This check causes issues. A link with mass can have no origin-translation for the inertia. We already check if the link has mass. That is enough.

Reply:
I am not sure to understand your point, we are using later `link.inertia.position`, so if it doesn't exist it will cause issues too, no?

Reply:
we dont check whether it exists, we check whether it is not [0, 0, 0]
Many urdf files have at least one link, that has a [0, 0, 0] translation for its inertia. In this case, we don't write the center of mass and Webots complains. As soon as we have a defined mass, we have to have a center of mass. We are already checking if we have mass, so this must be included, without additional checks.

Reply:
This adds useless center of mass new lines in the test (which is probably the cause of the failure):
```
centerOfMass [ 0.000000 0.000000 0.000000 ]
```

Reply:
We have to think about this critically!